src/btree/bt_slvg.c

```diff
@@ -157,11 +157,77 @@ static int  __slvg_trk_ovfl(WT_SESSION_IMPL *,
 		const WT_PAGE_HEADER *, uint8_t *, size_t, WT_STUFF *);
 
 /*
- * __wt_bt_salvage --
+ * __slvg_checkpoint --
+ *	Create the post-salvage checkpoint.
+ */
+static int
+__slvg_checkpoint(WT_SESSION_IMPL *session, WT_REF *root)
+{
+	WT_BTREE *btree;
+	WT_CKPT *ckptbase;
+	WT_DATA_HANDLE *dhandle;
+	WT_DECL_RET;
+	char *config;
+
+	btree = S2BT(session);
+	dhandle = session->dhandle;
+	config = NULL;
+
+	/*
+	 * XXX
+	 * The salvage process reads and discards previous checkpoints, so the
+	 * underlying block manager has to ignore any previous checkpoint
+	 * entries when creating a new checkpoint, in other words, we can't use
+	 * the metadata checkpoint list, it has all of those checkpoint listed
```

Comment:
I think that should be `all of those checkpoints...`

Reply:
Reworded for clarity in c55b7c2.